api/src/main/java/uk/gov/dvsa/mot/persist/jdbc/VehicleResultSetExtractor.java

```diff
@@ -75,6 +83,9 @@
                     }
                     motTest.setOdometerValue(String.valueOf(rs.getInt(15)));
                     motTest.setOdometerUnit(rs.getString(16));
+                    if (rs.getString(17) != null) {
```

Comment:
Magic number, extract into constant please

Reply:
I agree that constants would be better here. I refactored for this class (adding column indexes to constants and using them). However, these magic numbers exist throughout the entire project. It would be better for us to use column names (so we can edit the SQL without worrying about column orders);